### ISSUE TITLE
Remove test scope packages from resulting pom

### ISSUE THREAD

**Issue by rybalkinsd** [rybalkinsd/kohttp#18]
Problem with `gradle.build.kts`

for now https://search.maven.org/artifact/io.github.rybalkinsd/kohttp/0.1.0/jar
contains

    <dependency>
      <groupId>org.jetbrains.kotlin</groupId>
      <artifactId>kotlin-test-junit</artifactId>
      <version>1.2.60</version>
      <scope>test</scope>
    </dependency>

**Comment by gokulchandra** [comment]
@rybalkinsd is this still valid? 
I published the lib to my local maven repo and there were no test scope packages.

```
Gradle version: 5.0
Kotlin DSL:   1.0.4
Kotlin:       1.3.10
Groovy:       2.5.4
```

**Comment by rybalkinsd** [comment]
@gokulchandra seems to be fixed in `0.6.0` 
thanks for notice
